### ISSUE TITLE
3.0.0-beta.2 does not work with 3.0.0+

### ISSUE THREAD

**Issue by andrewfan** [sir-dunxalot/ember-tooltips#242]
I tried both popovers and tooltips and they do not render block content 
```
  {{#ember-tooltip targetId="avatar-dropdown"}}
    Here is some more info
  {{/ember-tooltip}}
```
the code above renders only 
```
<div class="tooltip ember-popover ember-tooltip-effect-slide ember-tooltip-show" role="tooltip" style="margin: 10px 0px 0px; position: absolute; transform: translate3d(1150px, 56px, 0px); top: 0px; left: 0px; will-change: transform;" id="tooltip_1qr11nokzw" aria-hidden="false" x-placement="bottom">
  <div class="tooltip-arrow ember-tooltip-arrow" style="left: 10px;"></div>
  <div class="tooltip-inner" id="ember317-wormhole"><span></span></div>
</div>
```

**Comment by erichonkanen** [comment]
is anyone still working on this lib? looks like there hasn't been a commit since October and the 3.0 branch is buggy.. I just installed and it didn't install wormhole

**Comment by sir-dunxalot** [comment]
Yes, I'm working on the 3.0.0 branch - development has just been very slow recently due to personal time constraints. I am working with others to bring a more active maintainer to push through some of the issues we're all experiencing.

**Comment by knownasilya** [comment]
Looks like this error shows up:

> "Assertion Failed: When calling `warn` you must provide an `options` hash as the third parameter.  `options` should include an `id` property."

The culprit is https://github.com/sir-dunxalot/ember-tooltips/blob/remove-tether-dependency/addon/components/ember-tooltip-base.js#L134

I'm using Ember 3.4.1

**Comment by maxfierke** [comment]
Both of these issues are fixed in `remove-tether-dependency` now, and will be in 3.0.0-beta.3.

**Comment by maxfierke** [comment]
This is fixed in 3.0.0-beta.3. Thanks!
